homeassistant/components/plugwise/config_flow.py

```diff
@@ -44,12 +62,47 @@ class ConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
     VERSION = 1
     CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
 
+    def __init__(self):
+        """Initialize the Plugwise config flow."""
+        self.discovery_info = {}
+
+    async def _async_set_unique_id(self, unique_id):
```

Comment:
Just inline this function, you're not saving any lines ?

Reply:
Removed

Reply:
You're right, inlined.